src/elements/content-picker/selectionCellRenderer.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @flow
+ * @file Function to render the checkbox table cell
+ * @author Box
+ */
+
+import React from 'react';
+import Checkbox from '../../components/checkbox';
+import RadioButton from '../../components/radio/RadioButton';
+import isRowSelectable from './cellRendererHelper';
+
+export default (
+    onItemSelect: Function,
+    selectableType: string,
+    extensionsWhitelist: string[],
+    hasHitSelectionLimit: boolean,
+    isRadio: boolean,
+): Function => ({ rowData }: { rowData: BoxItem }) => {
+    const { name, selected = false } = rowData;
+    const Component = isRadio ? RadioButton : Checkbox;
+    const props = {
+        [isRadio ? 'isSelected' : 'isChecked']: selected,
+        hideLabel: true,
+        label: ((name: any): string),
```

Comment:
Do we need to provide flow types for name? Is that already covered by the BoxItem flow type?

Reply:
These are props that are given to the component as standard `input` props. I don't think it needs flow types, unless we are going to declare the flow types on each `Checkbox` and `Radio`